packages/design-system-docs/src/pages/components/IdleTimeout/_IdleTimeout.docs.scss

```diff
@@ -0,0 +1,31 @@
+/*
+Idle Timeout
+
+A component that counts down to the end of a session (behind the scenes) and shows a warning message at a set time.
+
+Style guide: components.idleTimeout
+*/
+
+/*
+`<IdleTimeout>`
+
+@react-example IdleTimeout.example.jsx
+
+@react-props IdleTimeout.tsx
+
+Style guide: components.idleTimeout.react
+*/
+
+/*
+---
+
+### Related patterns
+
+- [Dialog]({{root}}/components/dialog)
+
+### When to use
+
+- Use the IdleTimeout for authenticated sessions to force logout after a set amount of time
+
```

Comment:
I'm not sure what information is helpful here since this component is for a pretty specific use case